packages/server/tests/integration/v1/boards.spec.ts

```diff
@@ -108,20 +127,149 @@ describe("POST /api/v1/boards", () => {
     expect(response.body.code).toBe("NOT_ENOUGH_PERMISSION");
   });
 
-  // it("should create a board", async () => {
-  //   // seed users with "board:create permission"
-  //   const createUser = await database
-  //     .insert([
-  //       {
-  //         userId: uuid(),
-  //         email: "create-board@example.com",
-  //         password: hashPassword("strongPassword"),
-  //         username: "create-board",
-  //       },
-  //     ])
-  //     .into("users")
-  //     .returning(["userId"]);
-  // });
+  it("should create a board", async () => {
```

Comment:
Can you split the test case?

- should create board with default values
- should create board with custom values